djangocms_picture/templates/djangocms_picture/picture.html

```diff
@@ -0,0 +1,77 @@
+ {% load thumbnail %}
+
+{% if instance.get_link %}
```

Comment:
link should be a context variable. these attribute access and conditionals can get expensive